### ISSUE TITLE
Check spelling temporarily stops working after word added to dictionary

### ISSUE THREAD

**Issue by bitinerant** [zim-desktop-wiki/zim-desktop-wiki#765]
To reproduce:
* Type a sentence in Zim with 2 or more misspelled words
* Enable Check spelling (F7)
* Right-click on one of the misspelled words and select Add "..." to Dictionary

Result: the other misspelled word(s) in the sentence and elsewhere on the page are no longer marked as misspelled until the user navigates away from the page and back again.

**Comment by bitinerant** [comment]
I am using Zim 0.70-rc3 on Ubuntu 18.04

**Comment by jaap-karssenberg** [comment]
THis is behavior controlled by the spellcheck library used. Please check in the plugin menu which dependency you are using.

**Comment by bitinerant** [comment]
Dependencies
• gtkspellcheck - Failed
• gtkspell - OK

**Comment by jaap-karssenberg** [comment]
Try installing gtkspellcheck that might improve it.

Op ma 8 apr. 2019 21:28 schreef bitinerant <notifications@github.com>:

**Comment by bitinerant** [comment]
Thanks for following up. I ran `sudo apt install -y python3-gtkspellcheck` and restarted Zim. Now I can no longer reproduce this issue, so this is a good work-around for me. However, there still seems to be an issue in the `gtkspell` implementation. Also, I found something strange: the plugin now shows that `gtkspell` is not installed, even though I did not uninstall anything:

Dependencies
• gtkspellcheck - OK
• gtkspell - Failed
